demos/jans-tarp/src/options/userDetails.tsx

```diff
@@ -32,7 +33,7 @@ const UserDetails = ({ data, notifyOnDataChange }) => {
     const [errorMessage, setErrorMessage] = React.useState("")
     const [authzResult, setAuthzResult] = React.useState("")
     const [authzLogs, setAuthzLogs] = React.useState("")
-
+    const [logType, setLogType] = React.useState('Decision');
```

Comment:
is the value state case-sensitive? if it's not I suggest the state value using low-case to avoid typo